<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewKeyboardWidget</class>
 <widget class="QWidget" name="AddNewKeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cbxKeyboardType"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chxWireless">
     <property name="text">
      <string>Wireless</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chxLighting">
     <property name="text">
      <string>Lighting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chxNumpad">
     <property name="text">
      <string>Numpad</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
